test/spec/token.js

```diff
@@ -0,0 +1,29 @@
+define(function(require) {
+  var OktaAuth = require('OktaAuth');
+  var tokens = require('../util/tokens');
+
+  describe('token.decode', function () {
+
+    function setupSync() {
+      return new OktaAuth({ url: 'http://example.okta.com' });
+    }
+
+    it('correctly decodes a token', function () {
+      var oa = setupSync();
+      var decodedToken = oa.token.decode(tokens.unicodeToken);
+      expect(decodedToken).toDeepEqual(tokens.unicodeDecoded);
+    });
+
+    it('throws an error for a malformed token', function () {
+      var oa = setupSync();
+      try {
+        var res = oa.token.decode('malformedToken');
+        // Should never hit this
+        expect(res).toBeNull();
```

Comment:
nit: To be clearer, it's probably best to test something that we know will never be true, i.e. "expect(true).toBe(false)". In this case if the decode method does return null for this case, the test won't fail. Or use other suggestion of throwing an error that's not an AuthSdkError so it'll fail in the next block.